app/src/main/java/net/skyscanner/backpack/demo/data/Tokens.kt

```diff
@@ -0,0 +1,52 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.demo.data
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import net.skyscanner.backpack.compose.tokens.BpkBorderRadius
+import net.skyscanner.backpack.compose.tokens.BpkColor
+import net.skyscanner.backpack.compose.tokens.BpkColors
+import net.skyscanner.backpack.compose.tokens.BpkElevation
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import kotlin.reflect.KProperty1
+import kotlin.reflect.full.memberProperties
+import kotlin.reflect.full.starProjectedType
+
+val BpkSpacing.values: List<Token<Dp>>
+  get() = BpkSpacing.values()
+
+val BpkBorderRadius.values: List<Token<Dp>>
+  get() = BpkBorderRadius.values()
+
+val BpkElevation.values: List<Token<Dp>>
+  get() = BpkElevation.values()
+
+val BpkColor.values: List<Token<Color>>
+  get() = BpkColor.values()
+
+val BpkColors.values: List<Token<Color>>
+  get() = BpkColors::class.memberProperties.mapNotNull {
```

Comment:
colour is a bit of a special case as we also have other functions here. i tried adding the return type check to the generic function as it might be helpful for future proofing anyways, but the check doesn't work with generics and i couldn't find a generic way to do it